lib/components/Stack/Stack.tsx

```diff
@@ -21,39 +26,47 @@ const alignToDisplay = {
   right: 'flex',
 } as const;
 
-export const useStackItem = ({ align, component, space }: UseStackProps) => {
-  const styles = useStyles(styleRefs);
-
-  return useBoxStyles({
-    component,
-    className: styles.excludingLast,
-    paddingBottom: space,
+export const useStackItem = ({ align, space }: UseStackItemProps) =>
+  ({
+    paddingTop: space,
     // If we're aligned left across all screen sizes,
     // there's actually no alignment work to do.
     ...(align === 'left'
-      ? {}
-      : {
-          display: mapResponsiveProp(align, alignToDisplay),
+      ? ({ display: 'block' } as const)
```

Comment:
As discussed, let's default the display down in the call to `normaliseResponsiveProps`